pyproject.toml

```diff
@@ -25,3 +25,10 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry.extras]
 docs = ["linkml", "mkdocs-material"]
+
+[tool.codespell]
+# Ref: https://github.com/codespell-project/codespell#using-a-config-file
+skip = '.git*,*.pdf,*.lock'
+check-hidden = true
+ignore-regex = '\b(COMENT|EHR|LOD)\b'
```

Comment:
is this where to add exceptions?

Reply:
yes, if you would like "case-sensitivity" or some fancy regex. Otherwise, just uncomment/list in the line below.
Also url above points to how docs which also describe inline pragmas for "one of" hits, so you could just add to the line `codespell:ignore` in the comment and the line would be ignored.

Reply:
Thank you!